src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width, double height) {
+        super(color, width * height);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: rectangle, area: %f, width: %f, height: %f, color: %s\n",
```

Comment:
Use only eng in messages/code: The output message should be in English. The word 'Figure' is already in English, but make sure that all parts of the message, including dynamic data, adhere to this rule.